vortex-array/src/aliases/mod.rs

```diff
@@ -1,2 +1,4 @@
 pub mod hash_map;
 pub mod hash_set;
+
+pub use hashbrown::DefaultHashBuilder as FoldHashBuilder;
```

Comment:
You could just call it the DefaultHashBuilder since the underlying hasher can change if we upgrade hashbrown